pkg/services/sqlstore/migrator/postgres_dialect.go

```diff
@@ -162,8 +162,8 @@ func (db *Postgres) PostInsertId(table string, sess *xorm.Session) error {
 	}
 
 	// sync primary key sequence of org table
-	if _, err := sess.Exec("SELECT setval('org_id_seq', (SELECT max(id)+1 FROM org));"); err != nil {
-		return fmt.Errorf("Failed to sync primary key for org table")
+	if _, err := sess.Exec("SELECT setval('org_id_seq', (SELECT max(id) FROM org));"); err != nil {
+		return fmt.Errorf("Failed to sync primary key for org table: %v", err)
```

Comment:
I should've mentioned this before, but we use an error wrapping utility in Grafana:

```suggestion
		return errutil.Wrapf(err, "failed to sync primary key for org table")
```

I think the norm in Go (and Grafana?) is also to start errors with lowercase first letter, since they are often combined and don't form the whole message if you see what I mean.